docs/codeql/writing-codeql-queries/defining-the-results-of-a-query.rst

```diff
@@ -34,78 +34,70 @@ If you look at some of the LGTM queries, you'll see that they can select extra e
 
 .. code-block:: ql
 
-   select f, "This file is similar to " + other.getBaseName()
+   select c, "This class extends the class " + superclass.getName()
 
-#. Element: ``f`` as before.
-#. String message: ``"This file is similar to "``—the string text is combined with the file name for the ``other``, similar file, returned by ``getBaseName()``.
+#. Element: ``c`` as before.
+#. String message: ``"This class extends the class "``—the string text is combined with the class name for the ``superclass``, returned by ``getName()``.
 
-.. image:: ../images/ql-select-statement-filename.png
+.. image:: ../images/ql-select-statement-class-name.png
    :alt: Results of extended select statement
    :class: border
 
-While this is more informative than the original select statement, the user still needs to find the other file manually.
+While this is more informative than the original select statement, the user still needs to find the superclass manually.
 
-Adding a link to the similar file
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Adding a link to the superclass
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You can use placeholders in the text of alert messages to insert additional information, such as links to the similar file. Placeholders are defined using ``$@``, and filled using the information in the next two columns of the select statement. For example, this select statement returns four columns:
 
 .. code-block:: ql
 
-   select f, "This file is similar to $@.", other, other.getBaseName()
+   select c, "This class extends the class $@.", superclass, superclass.getName()
 
-#. Element: ``f`` as before.
-#. String message: ``"This file is similar to $@."``—the string text now includes a placeholder, which will display the combined content of the next two columns.
-#. Element for placeholder: ``other`` corresponds to the similar file.
-#. String text for placeholder: the short file name returned by ``other.getBaseName()``.
+#. Element: ``c`` as before.
+#. String message: ``"This class extends the class $@."``—the string text now includes a placeholder, which will display the combined content of the next two columns.
+#. Element for placeholder: the ``superclass``.
+#. String text for placeholder: the class name returned by ``superclass.getBaseName()``.
 
-When the alert message is displayed, the ``$@`` placeholder is replaced by a link created from the contents of the third and fourth columns defined by the ``select`` statement.
+When the alert message is displayed, the ``$@`` placeholder is replaced by a link created from the contents of the third and fourth columns defined by the ``select`` statement. In this example, the link target will be the location of the superclass's definition, and the link text will be its name. Note that some superclasses, such as `Object``, will not be in the database, since they are built in to the Java language. Clicking those links will have no effect.
```

Comment:
```suggestion
When the alert message is displayed, the ``$@`` placeholder is replaced by a link created from the contents of the third and fourth columns defined by the ``select`` statement. In this example, the link target will be the location of the superclass's definition, and the link text will be its name. Note that some superclasses, such as ``Object``, will not be in the database, since they are built in to the Java language. Clicking those links will have no effect.
```